clean/tangipahoa_da.py

```diff
@@ -24,12 +30,15 @@ def clean():
         pd.read_csv(data_file_path("raw/tangipahoa_da/tangipahoa_da_brady_2021.csv"))
         .pipe(clean_column_names)
         .pipe(split_name)
+        .pipe(set_values, {"agency": "Tangipahoa SO", "source_agency": "Tangipahoa DA"})
         .pipe(assign_agency)
         .pipe(gen_uid, ["first_name", "last_name", "agency"])
+        .pipe(gen_uid, ["uid", "source_agency"], "allegation_uid")
+        .pipe(gen_uid, ["uid", "source_agency"], "brady_uid")
```

Comment:
Here, the `allegation_uid` is exactly the same as `source_agency`. Is there no other field we can rely on? If not perhaps we can generate the `allegation_uid` from `uid` and `agency` instead.